src/DAL/Events/Repositories/EventRepository.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace DAL.Events.Repositories
+{
+	sealed class EventRepository : IGenericRepository<Event, int>
+	{
+		private readonly EventContext _context;
+
+		public EventRepository(EventContext context)
+		{
+			_context = context;
+		}
+
+		public void Create(Event entity)
+		{
+			_context.Events.Add(entity);
```

Comment:
Could you explain benefits of having this method "void"?

Reply:
Passed entity would already be tracked and have Id on successful creation. Also returning `EntityEntry<T>` would be a disclosure of DAL. I didn't find anything real useful for return. Is there something?

Reply:
I mean why not using async version like in other methods.

Reply:
Oh, I didn't expect `AddAsync` to exist because `Add` method is a simple 'add tracking' operation. `AddAsync` can generate ids on client side. It should decrease latency of inserts.